### DISCUSSION TITLE
Lost password recovery [Aug 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6243] [Discord Threads]
> 🙋‍♂️ **productivix**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@771719010261532683> Will discuss with engineering team and get back on this  
#  
> 👨‍💻 **pranavxc**  
#### If you want to reset password of super admin you can setup environment variable, `NC_ADMIN_PASSWORD` and `NC_ADMIN_EMAIL`   
####    
#### https://docs.nocodb.com/getting-started/environment-variables  
#  
> 👨‍💻 **pranavxc**  
#### And for non-admin user you can use forgot password in  ui and then using `reset_password_token` value from db you can update the password by visiting `/auth/password/reset/${token}`  
#  
> 👨‍💻 **pranavxc**  
#### But for this you need DB access  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I did the super admin. I go then to my account and from there i can manage the other users access. I send them a link to reset their password. It was a huge improvement.  
#  
> 🙋‍♂️ **productivix**  
#### Yes I did that already : that runs perfect : I mean reset admin password  
#  
> 🙋‍♂️ **productivix**  
#### But I do not understand or know how to : " for non-admin user you can use forgot password in  ui and then using `reset_password_token` value from db you can update the password by visiting `/auth/password/reset/${token}`" can you please elaborate ?  thank you per advance  
#  
> 🙋‍♂️ **productivix**  
#### yes I super admin, I can sort of manage the users but some yes and others no : see the screen copy : 1st and last viewer : the fist have no hamburger menu and I cannot give him any link and tha last I can / I do not know how I have managed to have that situation, probably try to supress  the user that does not suppress by the way ... do you have a mean to cover the login aspect more comfortable for admin ?  
![image0](https://cdn.discordapp.com/attachments/1124356421232885831/1126064127358554123/Capture_decran_manage_links.png)  
#  
> 🙋‍♂️ **productivix**  
#### see a mean to solve around that issue, with my words of explanations : https://discord.com/channels/661905455894888490/1124302598556631161/1126068669605224572  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### <@771719010261532683>  this is where if you still need it.  
![image0](https://cdn.discordapp.com/attachments/1124356421232885831/1131599763368837148/image.png)  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### from your super admin account you will be able to manage other users  
#  
> 🙋‍♂️ **nachteulen.coffee**  
####   
![image0](https://cdn.discordapp.com/attachments/1124356421232885831/1131600036673900595/image.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1124356421232885831)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
